### ISSUE TITLE
Support editing annotation to confirm mappings

### ISSUE THREAD

**Issue by nichtich** [gbv/cocoda#470]
If a jskos-server instance is configured to support mappings of type `moderating` (confirmation of mappings) and if the current user is allowed to create/remove these annotations, the Cocoda user interface should support them. This requires https://github.com/gbv/jskos-server/issues/48. If both voting and confirmation are supported, we likely need another column in the mapping table.

**Comment by stefandesu** [comment]
My suggestion would have been to integrate the "confirmation" button into the current annotation popover and either show the score (if there is no annotation of type `moderating`) or show the confirmed arrow (if it was moderated). This is how it is currently implemented.

**Comment by stefandesu** [comment]
Backend support is now added with a (temporary) implementation using the config key `config.annotations.moderatingIdentities`. This should be used in Cocoda as well.

**Comment by stefandesu** [comment]
I added a first implementation, please test it out! I'm sure there will be some feedback, so I'll keep this issue open for now.

**Comment by nichtich** [comment]
Great! Minor UI issues: The "remove confirmation" button could better be a trashcan icon right to the corresponding confirmation in the popup. The color should not be hardcoded but taken from the same CSS class like uppvoted mappings. We likely need more features around confirmed mappings but only after it has been made clear who will actually confirm mappings.

**Comment by stefandesu** [comment]
> Minor UI issues: The "remove confirmation" button could better be a trashcan icon right to the corresponding confirmation in the popup.

For some reason, I blanked yesterday and couldn't think of that.

> The color should not be hardcoded but taken from the same CSS class like uppvoted mappings.

There are no hardcoded colors. Both the scores and the confirmation button use the same bootstrap classes.

Edit: I see which ones you mean, the color for the check icon. I will replace it with said bootstrap class. 

> We likely need more features around confirmed mappings but only after it has been made clear who will actually confirm mappings.

Of course, but that should be a separate issue I think.

**Comment by stefandesu** [comment]
Please take a look again. 👍
